### ISSUE TITLE
webpack-cli error

### ISSUE THREAD

**Issue by mroumanos** [jupyter-widgets/widget-ts-cookiecutter#33]
I'm seeing an error similar to https://github.com/webpack/webpack-cli/issues/607. I suggest updating version to 3+, as recommended in the issue.

**Comment by vidartf** [comment]
I'm working on a full overhaul which will fix this as well. It's currently blocked on another issue. If that takes to long to fix I'll implement a workaround :)
